Makefile.dune

```diff
@@ -4,7 +4,7 @@
 .PHONY: help voboot states world watch check          # Main developer targets
 .PHONY: coq coqide coqide-server                      # Package targets
 .PHONY: quickbyte quickopt                            # Partial / quick developer targets
-.PHONY: test-suite refman-html apidoc release         # Accesory targets
+.PHONY: test-suite refman-html stdlib-html apidoc release # Accesory targets
```

Comment:
Probably to split this line in two, including one for Documentation targets.

Reply:
Indeed.